provisionerd/provisionerd.go

```diff
@@ -367,6 +367,7 @@ func (p *Server) acquireAndRunOne(client proto.DRPCProvisionerDaemonClient) {
 			slog.F("workspace_build_id", build.WorkspaceBuildId),
 			slog.F("workspace_id", build.Metadata.WorkspaceId),
 			slog.F("workspace_name", build.WorkspaceName),
+			slog.F("is_prebuild", build.Metadata.IsPrebuild),
```

Comment:
nit: IMHO you can skip is_, just `prebuild` or `workspace_prebuild`. I haven't seen appending `is_` before.

Reply:
done

Reply:
@SasSwart I'm starting to think we're missing some changes?